### PR TITLE
Default to nanosecond clock time with GPL nvidia driver

### PR DESCRIPTION
**Pull request by ad1rie1**
Hello, 

Get nanosecond présentation timestamp now nvidia is a GPL driver

Dear,

### CODE REVIEW

**Comment by ValZapod** [comment]
Of god! BTW, windows recently also started exposing ns time (windows 7?), hope those are using ns.

**Comment by mtijanic** [comment]
Hello, welcome and thank you for your interest. As you're probably aware, this file is part of the nvidia-uvm.ko driver that has already been published as dual GPL/MIT. This driver is used in conjunction with both the newly published open nvidia.ko as well as traditional closed source version.

Applying this patch would therefore break the closed source driver which cannot depend on the GPL symbols. If I'm reading the code correctly, on the new driver we are already using the nanosecond precision and the fallback only applies to the proprietary module. 
In other words, the issue is with the 'dead' code and not functional. Is my understanding here correct?

**Comment by ad1rie1** [comment]
yes

**Comment by ValZapod** [comment]
Wait, wait! You can just do a #define! Obviously!

**Comment by ad1rie1** [comment]
@ValZapod the new driver already uses nanosecond precision as default, it's a mistake of myself.

**Comment by ValZapod** [comment]
So there is a completly different behaviour in closed source driver that still uses microseconds? Wow, that is even more horrible :)
